### PR TITLE
bug/ch103251/kyle-se-here-isolines-analysis-fails-due

### PR DESCRIPTION
**Pull request by juanrmn**
Allow isolines services for google-geocoding users.

The discussion here: https://app.clubhouse.io/cartoteam/story/103251/kyle-se-here-isolines-analysis-fails-due-to-google-geocoder

### CODE REVIEW

File: server/extension/cdb_dataservices_server--0.39.0--0.39.1.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+--DO NOT MODIFY THIS FILE, IT IS GENERATED AUTOMATICALLY FROM SOURCES
+-- Complain if script is sourced in psql, rather than via CREATE EXTENSION
+\echo Use "ALTER EXTENSION cdb_dataservices_server UPDATE TO '0.39.1'" to load this file. \quit
+
+-- HERE goes your code to upgrade/downgrade
```
Comment (Algunenano) [inline server/extension/cdb_dataservices_server--0.39.0--0.39.1.sql:5]: You also need to edit these files to include the upgrade and downgrade path. In the upgrade you (this file) you only need to incllude the functions/types that change between 0.39.0 and 0.39.1, and the other way around for the downgrade file. See old_versions/cdb_dataservices_server--0.37.0--0.38.0.sql // old_versions/cdb_dataservices_server--0.38.0--0.37.0.sql for an example.

**Review by Algunenano — APPROVED** [review]

**Review by Algunenano — APPROVED** [review]
LGTM. Please remember to also update the NEWs file before releasing/tagging a new version

**Review by juanrmn — COMMENTED** [review]
retest this, please

**Comment by juanrmn** [comment]
retest this, please

**Comment by juanrmn** [comment]
> LGTM. Please remember to also update the NEWs file before releasing/tagging a new version

Done! Thank you very much @Algunenano!

**Pull request merged**